tests/Microsoft.Identity.Web.Test.LabInfrastructure/UserInformationFieldIds.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Web.Test.Common;
+
+namespace Microsoft.Identity.Web.Test.LabInfrastructure
```

Comment:
modified version from MSAL. they have to check ADFS & B2C. we can add B2C later, so should be easy to expand this class when needed.